test/package.spec.ts

```diff
@@ -51,4 +54,28 @@ describe('Package', () => {
     expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['package.json', 'package-lock.json', 'node_modules']);
     expect(fs.readdirSync(path.resolve(projectDir, 'deployment', 'node_modules', '@nestjs'))).not.toContain('cli');
   }, 60000);
+
+  it('should not show warning messages when old dependencies are not used', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'without-dependencies');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+
+    await Package.run([projectDir, '--skipInstall']);
+
+    expect(boxMessage).toBeCalledWith(expect.arrayContaining(['✅ Package finished successfully.']));
```

Comment:
Minor: Perhaps you can also add that it does not contain the ` Old SAP Cloud SDK: @sap/cloud-sdk-core is detected.', 'Please find how to migrate here:`